lokerpc/codegen/typescript.go

```diff
@@ -97,7 +97,7 @@ func GenTypescriptClient(w io.Writer, meta lokerpc.Meta) error {
 
 	b.WriteString("\n")
 	tsDocComment(b, meta.Help, "")
-	b.WriteString("export class " + capitalize(meta.ServiceName) + "Service extends RPCContextClient {\n")
+	b.WriteString("export class " + pascalCase(meta.ServiceName) + "Service extends RPCContextClient {\n")
```

Comment:
Sorry, mustn't have been clear, we never have `-` in service names, but object keys might, probably want to quote keys if they have any non valid char. Maybe `!/[a-z_$][a-z0-9_$]*/.test(k)`

Reply:
`stripe-payments` is hyphenated.

Don't think we can change that without breaking things.

Reply:
Okay leave it then, but still should add the better quoting rules

Reply:
- oauth-clients
- sms-verifications
- aston-customers
- report-data
- global-auth-clients
- global-auth-users
- stripe-payments

Must be close to half of the services that are hyphenated

Reply:
> add the better quoting rules

ok, will do

Reply:
Ah okay, must have changed our convention at some point, probably when all services moved to having a path

Reply:
Actually probably need to match the start and end too `!/^[a-z_$][a-z0-9_$]*$/.test(k)`

Reply:
Not sure about this convention, feel like `aston-customers` has been around for years 🤷

Reply:
Haha okay maybe I'm just completely wrong :P

Reply:
Hmm to be fair, those aren't the service names, these are
```
appmenu
apps
appstats
customers
builds
checkins
credits
customers
discounts
clients
users
inventory
legacyapps
locations
merchants
notes
notifications
oauth-clients
order
orders
orders2
orders3
organizationadmins
organizations
organizationsettings
payattable
paymentoptions
points
profilefields
report-data
sms-verifications
stripe-payments
themes
users
```
`global-auth-clients` is just the file name of the manifest

Reply:
Ah ok so most are just hyphenated file names not service names.

Maybe we can fix up naming later but we need to expose over the old and new name to transition. Don't think it is worth the time investment now. Is there anything wrong with just supporting it?

Reply:
Also... we have way more services than I thought!

Reply:
Yeah probably better if we switch to kebab anyway, better for codegen